src/makeCalculator.js

```diff
@@ -36,8 +36,47 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
+
+    subtract: function(num) {
+      calculator.result -= num;
+
+      return calculator;
+    },
+
+    multiply: function(num) {
+      calculator.result *= num;
+
+      return calculator;
+    },
+
+    divide: function(num) {
+      calculator.result /= num;
+
+      return calculator;
+    },
+
+    reset: function() {
+      calculator.result = 0;
+
+      return calculator;
+    },
```

Comment:
You should use `this` instead of calculator reference
```suggestion
    add: function(num) {
      this.result += num;

      return this;
    },

    subtract: function(num) {
      this.result -= num;

      return this;
    },

    multiply: function(num) {
      this.result *= num;

      return this;
    },

    divide: function(num) {
      this.result /= num;

      return this;
    },

    reset: function() {
      this.result = 0;

      return this;
    },
```

Reply:
Ok